src/p2p/Join/index.ts

```diff
@@ -251,17 +259,35 @@ export function updateRecord(txs: P2P.JoinTypes.Txs, record: P2P.CycleCreatorTyp
 
   if (config.p2p.useJoinProtocolV2) {
     // for join v2, add new standby nodes to the standbyAdd field ...
-    for (const standbyNode of drainNewJoinRequests()) {
-      record.standbyAdd.push(standbyNode)
+    for (const request of txs.standbyAdd) {
+      const publicKey = request.sign.owner
+      const node = NodeList.byPubKey.get(publicKey)
```

Comment:
This line also wont work for standbyAdd. these nodes are unknow to any of the nodelists if they arent even on the standby list. so we wont be able to do a check at all